plugins/woocommerce-blocks/assets/js/blocks/product-filters/edit.tsx

```diff
@@ -62,6 +75,30 @@ const TEMPLATE: InnerBlockTemplate[] = [
 			heading: __( 'Rating', 'woocommerce' ),
 		},
 	],
+	[
+		'core/buttons',
+		{ layout: { type: 'flex' } },
+		[
+			[
+				'core/button',
+				{
+					text: __( 'Apply', 'woocommerce' ),
+					className: 'wc-block-product-filters__apply-button',
+					style: {
+						border: {
+							width: '0px',
+							style: 'none',
+						},
+						typography: {
+							textDecoration: 'none',
+						},
+						outline: 'none',
+						fontSize: 'medium',
+					},
+				},
+			],
+		],
+	],
```

Comment:
Not sure about the Apply button as I think it doesn't relate to overlay setting as commented [here](https://github.com/woocommerce/woocommerce/pull/48732#pullrequestreview-2146953604). But we can change its behavior later so let's keep it for now.